sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Features Added
 
 ### Bugs Fixed
+- Removed `requests` import from `azure.storage.blob._download`.  #25017
+- Fixed retry mechanism in `azure.storage.blob._download`.  #25017
```

Comment:
nit: we don't want to mention private modules, so it can be like 'Removed forced `requests` import for sync calls' and 'fixed blob download retry mechanism' or something like that

Reply:
Super nit: Same feedback as other PR, see line 68 (original) or line 70 (your current commit) for an example of mentioning an issue in the changelog for consistency sake 😄